### PR TITLE
chore: update grav schema

### PR DESCRIPTION
**Pull request by mdole**
Pulls in new artistIDs array types for Search Criteria, following up on https://github.com/artsy/gravity/pull/14910/files

### CODE REVIEW

File: src/data/gravity.graphql [hunk 1]
```diff
@@ -1919,7 +1922,12 @@ Attributes for a search criteria
 input SearchCriteriaAttributes {
   acquireable: Boolean = null
   additionalGeneIDs: [String!] = []
+
+  """
+  DEPRECATED Please use artistIDs instead
+  """
```
Comment (gkartalis) [inline src/data/gravity.graphql:1928]: 🤩

File: src/data/gravity.graphql [hunk 2]
```diff
@@ -1055,9 +1055,11 @@ type LotEvent {
   createdAt: String!
   internalID: ID!
   key: String!
+  paddleNumber: String
   type: String!
   userAgent: String
   userID: ID
+  userName: String
```
Comment (gkartalis) [inline src/data/gravity.graphql:1062]: I don't get why this change is included 🤔

**Review by gkartalis — DISMISSED** [review]

**Review by gkartalis — APPROVED** [review]
updated the snapshot tests and pushed, looking good

File: src/data/gravity.graphql (hunk 2 above)
Comment (mdole) [reply src/data/gravity.graphql:1062]: just because someone else made updates to the Gravity schema and didn't pull them in to Metaphysics (I think Ohm may be referencing GravQL directly)
